include/tvm/relay/attrs/transform.h

```diff
@@ -75,10 +75,15 @@ struct ReshapeAttrs : public tvm::AttrsNode<ReshapeAttrs> {
 
 struct TakeAttrs : public tvm::AttrsNode<TakeAttrs> {
   Integer axis;
+  std::string mode;
 
   TVM_DECLARE_ATTRS(TakeAttrs, "relay.attrs.TakeAttrs") {
     TVM_ATTR_FIELD(axis).set_default(NullValue<Integer>())
         .describe("The axis over which to select values.");
+    TVM_ATTR_FIELD(mode).set_default("CLIP")
+        .describe("Specify how out-of-bound indices will behave."
+                  "CLIP - clip to the range (default)"
+                  "WRAP - wrap around the indices");
```

Comment:
suggest to use lower-case, to be the same as that in numpy. also add `mode="raise"` to the description

Reply:
Sure. I can change to lower case. 
It's a bit tricky to add "raise" mode because current tvm::compute (see https://github.com/dmlc/tvm/blob/master/include/tvm/operation.h#L547) doesn't allow us to add "assert" stmt in the body. I think we should add this in a follow-up pr.

Reply:
how about we skip raise for now?